packages/common/test/utils.test.ts

```diff
@@ -0,0 +1,34 @@
+import { getPackageJsonTasks } from '../src/utils';
+
+describe('Test common utils', () => {
```

Comment:
There are 3 tests inside this `test()` method. Would it be possible to split them into 3 and add descriptive comments so that it's clear which usecase is being tested?

Reply:
Not sure, there are no test cases per se but just various combinations of parameters that should be tested at the very least.

Reply:
They are testing that the outputs of `getPackageJsonTasks` are as expected based on those inputs. Im not sure I understand what the ask is here?